ios/22/_sentry_arm64.crash

```diff
@@ -0,0 +1,35 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BREAKPOINT
+Crashed Thread: 0
+
+Application Specific Information:
+Exception 6, Code 771328, Subcode 8
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x1000bc500         @objc CRLCrashSwift.crash() -> ()
+1   CrashLibiOS                     0x1000bc500         CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:36)
```

Comment:
Our goal was to get as close as possible to the backtrace you get when you see the crash in xcode/lldb. The way to do that is to also handle inline frames from the DWARF data.

This is indeed intentional. The crash reporting library is only producing one frame which with the data contained in the dsym DWARF data can expand to multiple. In this case the apple conpiler stores an inline function call in the debug data and we support that. This is consistent with backtraces generated by xcode/lldb. We do want to keep this feature enabled because it gives signficiantly better stacktraces at higher optimization levels in case the compiler inlines small functions.